<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="304" started-at="2011-07-29T18:01:44Z" finished-at="2011-07-29T18:01:44Z">
    <groups>
      <group name="mock">
        <method signature="org.exoplatform.social.client.core.service.ActivityServiceTest.testCreateActivity()" name="testCreateActivity" class="org.exoplatform.social.client.core.service.ActivityServiceTest"/>
      </group>
    </groups>
    <test name="Default test" duration-ms="304" started-at="2011-07-29T18:01:44Z" finished-at="2011-07-29T18:01:44Z">
      <class name="org.exoplatform.social.client.core.service.ActivityServiceTest">
        <test-method status="PASS" signature="setUp()" name="setUp" is-config="true" duration-ms="271" started-at="2011-07-29T18:01:44Z" finished-at="2011-07-29T18:01:44Z">
        </test-method>
        <test-method status="FAIL" signature="testCreateActivity()" name="testCreateActivity" duration-ms="18" started-at="2011-07-29T18:01:44Z" finished-at="2011-07-29T18:01:44Z">
          <exception class="java.lang.IllegalStateException">
            <message>
              <![CDATA[missing behavior definition for the preceding method call create(EasyMock for class org.exoplatform.social.client.core.model.RestActivityImpl)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalStateException: missing behavior definition for the preceding method call create(EasyMock for class org.exoplatform.social.client.core.model.RestActivityImpl)
	at org.easymock.internal.MocksControl.replay(MocksControl.java:174)
	at org.easymock.EasyMock.replay(EasyMock.java:1970)
	at org.exoplatform.social.client.core.service.ActivityServiceTest.testCreateActivity(ActivityServiceTest.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:685)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:854)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1178)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1182)
	at org.testng.TestRunner.privateRun(TestRunner.java:761)
	at org.testng.TestRunner.run(TestRunner.java:612)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:292)
	at org.testng.SuiteRunner.run(SuiteRunner.java:241)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1169)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1094)
	at org.testng.TestNG.run(TestNG.java:1006)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:107)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:199)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:170)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
